truss/templates/server/common/util.py

```diff
@@ -12,10 +17,19 @@ def model_supports_predict_proba(model):
     return True
 
 
-def assign_request_to_inputs_instances_after_validation(request):
+def assign_request_to_inputs_instances_after_validation(body: dict) -> dict:
     # we will treat "instances" and "inputs" the same
-    if "instances" in request and "inputs" not in request:
-        request["inputs"] = request["instances"]
-    elif "inputs" in request and "instances" not in request:
-        request["instances"] = request["inputs"]
-    return request
+    if "instances" in body and "inputs" not in body:
+        body["inputs"] = body["instances"]
+    elif "inputs" in body and "instances" not in body:
+        body["instances"] = body["inputs"]
+    return body
+
+
+def setup_logging():
```

Comment:
does this change how logs will look when the server is running?

Reply:
This should be functionally the same as what was in `configure_logging` before, just moved and simplified. Shouldn't have any change. In tests, logging looked same as previous.

Reply:
Of note, lets keep our eyes out as this area is infamously untested.